docs/how-to-add-support-for-new-package-manager.md

```diff
@@ -0,0 +1,43 @@
+# How to add support for a new package manager
+
+The sections below outline the steps needed to add support for a new package manager to OSS Review Toolkit.
+
+## 1. Initial Questions / Requirements
+
+Before you start any implementation work, determine whether the package manager meets the minimal requirements to provide the data needed for OSS Review Toolkit to work.
+
+Answers to the questions below should help you 
```

Comment:
You should mention that the examples below are for checking if pnpm support could be added, otherwise it's confusing that the first point talks of pnpm.

Also, all points below are not questions, they should be rephrased:

1. How to detect whether a project is a pnpm project?
2. How to get the declared license of a package?
...

Reply:
done